src/runtime/entitlements-manager.js

```diff
@@ -375,6 +376,7 @@ export class EntitlementsManager {
     }
     const signedData = json['signedEntitlements'];
     const decryptedDocumentKey = json['decryptedDocumentKey'];
+    const swgUserToken = json['swgUserToken'];
```

Comment:
Will this be sent back in the entitlement response as sut or swgUserToken? Should we keep the naming consistent across the board, even in the js code?

Reply:
sut is the entitlement request url param. swgUserToken is in the entitlement response from boq.